tox.ini

```diff
@@ -84,4 +84,14 @@ deps=-r{toxinidir}/lib/rnaget/rnaget_service/requirements.txt
 
 commands=
     python setup.py develop
-    screen -dmS {envname} candig_rnaget --host={env:RNA_GET_HOST} --port={env:RNA_GET_PORT}
\ No newline at end of file
+    screen -dmS {envname} candig_rnaget --host={env:RNA_GET_HOST} --port={env:RNA_GET_PORT}
+
+[testenv:datasets]
+basepython=python3.7
```

Comment:
Just curious, why is python3.7 needed here if we are already using it for global?

Reply:
Fixed.